### PR TITLE
working functionality for datepicker min and max disable functionality.

### PR DESCRIPTION
**Pull request by randallkanna**
@pzuraq

### CODE REVIEW

File: app/styles/ticketfly-ui/components/_tf-date-picker.scss [hunk 1]
```diff
@@ -102,8 +102,12 @@
         display: block;
       }
 
+      &.isNotInMonth {
```
Comment (pzuraq) [inline app/styles/ticketfly-ui/components/_tf-date-picker.scss]: css class names should be dasherized, `.is-not-in-month`

File: app/styles/ticketfly-ui/components/_tf-date-picker.scss [hunk 2]
```diff
@@ -102,8 +102,12 @@
         display: block;
       }
 
+      &.is-not-in-month {
```
Comment (randallkanna) [inline app/styles/ticketfly-ui/components/_tf-date-picker.scss]: @pzuraq  This has been fixed and the template in date-picker and date-picker range has been modified.

File: addon/components/ui-date-range-picker.js [hunk 3]
```diff
@@ -110,26 +125,30 @@ export default DatePicker.extend({
...
-          set(this, 'startDate', date);
-          set(this, 'endDate', date);
-        } else {
-          if (moment(date).isBefore(startDate)) {
-            this.set('startDate', date);
-          } else {
+    selectDate(date, dropdown, isDisabled) {
+      var disabledDate = isDisabled;
+
+      if (!disabledDate) {
```
Comment (pzuraq) [inline addon/components/ui-date-range-picker.js]: So, a cleaner pattern to follow here is the following:

```
if (isDisabled) { return; }
```

This basically ends the function if isDisabled is true, without having to wrap everything inside the function in brackets.

File: addon/components/ui-date-range-picker.js [hunk 4]
```diff
@@ -110,26 +125,30 @@ export default DatePicker.extend({
...
-      if (this.get('range')) {
-        if (!startDate || !get(this, 'isSingleDate')) {
-          set(this, 'startDate', date);
-          set(this, 'endDate', date);
-        } else {
-          if (moment(date).isBefore(startDate)) {
-            this.set('startDate', date);
-          } else {
+    selectDate(date, dropdown, isDisabled) {
+      var disabledDate = isDisabled;
```
Comment (pzuraq) [inline addon/components/ui-date-range-picker.js]: Also no reason to create a new variable here, just use the one passed in.

Comment (randallkanna) [inline addon/components/ui-date-range-picker.js]: @pzuraq  Ok! Updated this to reflect the cleaner and refactored code.

File: addon/components/ui-date-range-picker.js (hunk 3 above)
Comment (randallkanna) [inline addon/components/ui-date-range-picker.js]: @pzuraq Makes much more sense! Fixed this.

**Pull request merged**